src/sentry/migrations/0348_add_snowflake_seq.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 2.2.28 on 2022-12-29 04:53
+
+from django.db import migrations
+
+import sentry.db.models.fields.bounded
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production. For
+    # the most part, this should only be used for operations where it's safe to run the migration
+    # after your code has deployed. So this should not be used for most operations that alter the
+    # schema of a table.
+    # Here are some things that make sense to mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that they can
+    #   be monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   have ops run this and not block the deploy. Note that while adding an index is a schema
+    #   change, it's completely safe to run the operation after the code has deployed.
+    is_dangerous = False
+
+    dependencies = [
+        ("sentry", "0347_add_project_has_minified_stack_trace_flag"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
```

Comment:
We don't technically _need_ a table for this at all. Does django require the table for sequences? If this is the easiest way to do this, np, but worth looking.

Reply:
It's, really obnoxious.  I looked into this earlier.  Basically,  if we create a sequence with a raw migration, *tests do not sync that migration* because it isn't applied to the "models state".  And Django's "models state" has no means to store sequences that are not auto fields.  The short answer is that we can have sequences without models, but they won't show up in tests.